File: Códigos em C/Verificador de Maioridade.c
#include <stdio.h>

int main()
{
    int idade;
    printf("Verificador de Maioridade");
    printf("\n\nDigite sua idade: ");
    scanf("%d",&idade);
    
    if (idade >= 18){
        printf("Você é maior de idade.");
    }
    else{
        printf("Você é menor de idade.");
    }
    return 0;
}

File: Códigos em Haskell/Verificador de Maioridade.hs
maioridade x = if x >= 18 then "Voce e maior de idade" else "Voce e menor de idade"

main = do
  putStrLn "Verificador de Maioridade"
  putStrLn ""
  putStrLn "Digite sua idade: "
  idade <- readLn :: IO Float
  putStrLn ""
  putStrLn (show(maioridade idade))
  
  
